src/api/common/utils.js

```diff
@@ -27,6 +28,11 @@ function toRippledAmount(amount: Amount): string|Amount {
   };
 }
 
+function generateWallet(options) {
```

Comment:
Yeah, I did at some point, but then it complained about module missing for `ripple-keypairs`. I'll look into it more.